### PR TITLE
Wip/class rename

### LINKED ISSUE #37: Rename client class to MarquezClient()

### PR DESCRIPTION
**Pull request by ashulmanWeWork**
Closes #37

### CODE REVIEW

**Comment by sshah-wework** [comment]
Ping?

**Comment by ashulmanWeWork** [comment]
@sshah-wework, this is an improvement that @wslulciuc and I wanted to get in to ease usability. Unfortunately we ran out of time and needed to focus on higher-priority items, so we decided to leave this here.

That being the case, it's ready to go, but while I'd like to see this get in, I realize it will also require some work updating marquez-airflow when @roaraya8 is ready to upgrade to this.

I think we can take this on when we circle back to updating the client with Willy's recent timestamp changes (among others).

**Review by wslulciuc — APPROVED** [review]
LGTM!

File: .circleci/config.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
       - run: pip install --user coverage
       - run: pip install --user -e .
       - run: pip install --user flake8
-      - run: python -m flake8
+      - run: python -m flake8 --extend-ignore=F401
```
Comment (wslulciuc) [inline .circleci/config.yml:17]: Why do we need to add this exception?

Comment (ashulmanWeWork) [reply .circleci/config.yml:17]: flake8 complains when we import something and do not use it. This is the case in the file we're adding, `marquez_client/__init__.py`.

**Pull request merged**